pandas/core/generic.py

```diff
@@ -5284,6 +5284,8 @@ def head(self: NDFrameT, n: int = 5) -> NDFrameT:
         For negative values of `n`, this function returns all rows except
         the last `n` rows, equivalent to ``df[:-n]``.
 
+        If n is larger than the number of rows, this function returns al rows.
```

Comment:
`al` -> `all`? Same as below

Reply:
You can just add another commit fixing this typo to this pull request. When merged, all the commits will be squashed.

```
# Fix the typo
git add pandas/core/generic.py
git commit -m"Fixed all typo"
git push origin <branch_name>
```